_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
I like the idea of always displaying the message (and your raw sketch was pretty good). People would feel more confident reading the translated file, knowing that it is up to date. I don't know about displaying number of changes, how could we do that?

What I think would be best is that it would always display the message, and if it is not green, it would make the box clickable linking to the repo showing the lates commits on the english translation for the file. The link would be like this: 
https://github.com/doomspork/elixir-school/commits/master/lessons/#{category}/#{lesson}
so that anyone who wants to contribute, can check on the latest changes of the file.